### DISCUSSION TITLE
Bottom Layer Optimization

### DISCUSSION THREAD

**Discussion by Trydnt** [OrcaSlicer/OrcaSlicer#7758] [Ideas]
Easy implementation: Expose all seam variables in height range modifier options so the first layer can be done differently from the rest of the object.

Medium implementation: Minimize travel distance over "empty space" that hasn't been filled in yet. This will reduce or eliminate issues of stringing across first layer print area when laying out perimeters, which frequently leads to poor adhesion, failed prints, and a lower quality finish on the bottom layer. (Related post - "How to Wipe Properly", in the bottom layer, any time when the nozzle is potentially leaking filament it should avoiding moving across any area that haven't already been printed on. These issues are related but separate. But together they'd make Orca the most powerful slicer imaginable)

Harder implementation: Manually set bottom layer loop print order and seams. This allows you to manually set the order for loops in the first layer AND the loop start/end points. This would be similar to the seam painting tool that already exists but only for the first layer. This will allow for perfect optimization of the first layer and lead to far more successful prints, however it is a more sophisticated add-on to the software than the other options.

**Comment by DARKFiB3R** [comment]
Please share an .stl of the first 2 layers of a model you are having problems with. I'd like to take a look at the issue.
